.buildkite/pipeline.yml

```diff
@@ -91,6 +99,21 @@ steps:
       distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
       hostname: "${BUILDKITE_AGENT_META_DATA_HOSTNAME}"
 
+  - label: ':hammer: test against firecracker master'
+    commands:
+      - 'ln -s /var/lib/fc-ci/vmlinux.bin testdata/vmlinux'
+      - 'ln -s /var/lib/fc-ci/rootfs.ext4 testdata/root-drive.img'
+      - 'cp /usr/local/bin/firecracker-v0.20.0 testdata/firecracker'
+      - 'cp /usr/local/bin/jailer-v0.20.0 testdata/jailer'
+      - 'make -C cni install CNI_BIN_ROOT=$(pwd)/testdata/bin'
+      - "sudo FC_TEST_TAP=fc-root-tap${BUILDKITE_BUILD_NUMBER} make test-in-docker EXTRAGOARGS='-v -count=1 -race' DISABLE_ROOT_TESTS="
```

Comment:
Building Firecracker on Docker makes sense, but testing Go SDK on Docker doesn't (for me).

It would be better to either 1) use Docker for 0.20.0 tests as well or 2) don't use Docker for running tests. We are introducing unnecessary differences between 0.20.0 tests and master tests.

Reply:
The reason why this was done is we'd need to swap out the binaries for the tests, one for master and another for the versioned. I'd rather avoid all of that and have it run in Docker.

Reply:
How about passing FC_TEST_BIN?

https://github.com/firecracker-microvm/firecracker-go-sdk/blob/57d08e35c3f176d2b9e8c43c220a3a6548c611ee/HACKING.md#L17

Reply:
Good call! Totally forgot about that. Will update.